tests/test_passwordless.py

```diff
@@ -99,3 +101,70 @@ async def send_text_message(param: passwordless.CreateAndSendCustomTextMessagePa
     consume_code_json['user'].pop('time_joined')
 
     assert consume_code_json == {'status': 'OK', 'createdNewUser': True, 'user': {'phoneNumber': '+919494949494'}}
+
```

Comment:
More test cases:
- delete email works
- if you delete phone and email of a user, an error is thrown
- Thirdpartypasswordless deleting email and phone works.